docs/process.md

```diff
@@ -41,105 +42,65 @@ Costs are
+
+### Exploratory Status
+
+When a formal draft has been discussed sufficiently, submitted to the PSC, and is under consideration, it is entered into the proposal tracker, with the status **Exploratory**.  The PSC (or other deputized folk) will begin the process of vetting the idea.  They will review discussion about the idea, they will produce a list of questions not yet answered, and they will press existing core team members (or other experts) for input.
+
+The PSC (or deputies) will eventually either:
+ * move the document to **Implementing** status (see below) because they believe it is ready for implementation
+ * move the document to **Rejected** status because they believe it should not be implemented; This may come with advice on how to formulate an alternate proposal that has more chance of being accepted.  This isn't used for "it needs some edits", but for "it's fundamentally deeply flawed."
+ * move the document to the **Expired** status because the original proposer (or other vital personnel) are not responsive
+
+During this phase, the PSC is responsible for moving the proposal forward.
+
+### Implementing
+
+When a proposal is accepted, the PSC is stating that they'd like to see the idea implemented, and that they are likely to merge an implementation if it doesn't bring with it any unwelcome surprises or complications.
+
+If no implementation has made progress for three months, the document moves to **Expired**.
+
+If an implementation exposes serious problems that mean the PSC no longer believes the proposal can work, the document moves to **Rejected**.
+
+If an implementation is delivered and it isn't clear that it's broken as designed, the document moves to **Testing** status.
+
+During this phase, the proposer or other named implementor is responsible for moving the proposal forward.  The PSC will only review proposals in this status when work is delivered or when status updates cease.  The PSC will make a regular note of proposals in this status to perl5-porters.  The implementors of proposals in this status will post regular updates -- regular enough, at any rate, to avoid becoming Expired.
+
+### Testing
+
+Now there's an accepted proposal and an implementation.  At this point, it needs to have enough tests, showing how it will work in real world conditions.  The PSC will consult with the author, each other, perl5-porters, and other experts to produce a list of needed test cases.  The proposer, implementer, or other volunteers will provide the testing.  Once enough testing is done, then the document will be either…
+
+ * marked **Accepted**, with the code merged as a new (probably experimental!) feature of perl
+ * marked **Rejected**, because the quality or behavior of the feature or its implementation are not acceptable
```

Comment:
is it possible to move an RFC that failed tests back into the "implementing" state? For example, let's say one of the tests fail on an exotic-yet-supported platform perl runs on, exposing an issue on the implementation that requires a big refactoring. Should it go back to "implementing" or stay "testing" during the fix?

Reply:
I expect a lot of the state-transition edges could allow backwards stepping, yes; if on closer inspection the overall idea is found to be less ready for that new state than it was first thought, then the correct thing to do is probably just to send it back again.